2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Configure stats pid to 8568
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Alexia\.config\wandb\settings
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Alexia\diffusion_ts\wandb\settings
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-22 10:20:25,903 INFO    MainThread:8568 [wandb_init.py:_log_setup():527] Logging user logs to c:\Users\Alexia\diffusion_ts\wandb\run-20240422_102025-7r6pm3st\logs\debug.log
2024-04-22 10:20:25,907 INFO    MainThread:8568 [wandb_init.py:_log_setup():528] Logging internal logs to c:\Users\Alexia\diffusion_ts\wandb\run-20240422_102025-7r6pm3st\logs\debug-internal.log
2024-04-22 10:20:25,907 INFO    MainThread:8568 [wandb_init.py:_jupyter_setup():473] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002005CBD5700>
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_init.py:init():567] calling init triggers
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_init.py:init():592] re-initializing run, found existing run on stack: izluv5t2
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_run.py:_finish():2052] finishing run fyp_a/test_train_2/izluv5t2
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_run.py:_restore():2275] restore
2024-04-22 10:20:25,908 INFO    MainThread:8568 [wandb_run.py:_restore():2281] restore done
2024-04-22 10:20:30,961 INFO    MainThread:8568 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-22 10:20:30,961 INFO    MainThread:8568 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-22 10:20:30,961 INFO    MainThread:8568 [wandb_run.py:_footer_sync_info():3879] logging synced files
2024-04-22 10:20:31,012 INFO    MainThread:8568 [wandb_init.py:init():617] starting backend
2024-04-22 10:20:31,012 INFO    MainThread:8568 [wandb_init.py:init():621] setting up manager
2024-04-22 10:20:31,015 INFO    MainThread:8568 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-22 10:20:31,015 INFO    MainThread:8568 [wandb_init.py:init():629] backend started and connected
2024-04-22 10:20:31,020 INFO    MainThread:8568 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-22 10:20:31,020 INFO    MainThread:8568 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-22 10:20:31,020 INFO    MainThread:8568 [wandb_init.py:init():721] updated telemetry
2024-04-22 10:20:31,049 INFO    MainThread:8568 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-04-22 10:20:32,681 INFO    MainThread:8568 [wandb_run.py:_on_init():2344] communicating current version
2024-04-22 10:20:33,060 INFO    MainThread:8568 [wandb_run.py:_on_init():2353] got version response upgrade_message: "wandb version 0.16.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-22 10:20:33,060 INFO    MainThread:8568 [wandb_init.py:init():805] starting run threads in backend
2024-04-22 10:20:33,270 INFO    MainThread:8568 [wandb_run.py:_console_start():2323] atexit reg
2024-04-22 10:20:33,270 INFO    MainThread:8568 [wandb_run.py:_redirect():2178] redirect: wrap_raw
2024-04-22 10:20:33,270 INFO    MainThread:8568 [wandb_run.py:_redirect():2243] Wrapping output streams.
2024-04-22 10:20:33,270 INFO    MainThread:8568 [wandb_run.py:_redirect():2268] Redirects installed.
2024-04-22 10:20:33,270 INFO    MainThread:8568 [wandb_init.py:init():848] run started, returning control to user process
2024-04-22 10:20:59,630 INFO    MainThread:8568 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-22 10:20:59,632 INFO    MainThread:8568 [wandb_init.py:_pause_backend():438] pausing backend
2024-04-22 10:26:43,815 INFO    MainThread:8568 [wandb_init.py:_resume_backend():443] resuming backend
2024-04-22 10:26:43,826 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-22 10:26:43,826 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Configure stats pid to 8568
2024-04-22 10:26:43,826 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Alexia\.config\wandb\settings
2024-04-22 10:26:43,826 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Alexia\diffusion_ts\wandb\settings
2024-04-22 10:26:43,826 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:_log_setup():527] Logging user logs to c:\Users\Alexia\diffusion_ts\wandb\run-20240422_102643-qragp0yg\logs\debug.log
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:_log_setup():528] Logging internal logs to c:\Users\Alexia\diffusion_ts\wandb\run-20240422_102643-qragp0yg\logs\debug-internal.log
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:init():567] calling init triggers
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:init():592] re-initializing run, found existing run on stack: 7r6pm3st
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_run.py:_finish():2052] finishing run fyp_a/test_train_2/7r6pm3st
2024-04-22 10:26:43,827 ERROR   MainThread:8568 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-22 10:26:43,827 INFO    MainThread:8568 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_init.py:_jupyter_teardown():455] cleaning up jupyter logic
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_run.py:_restore():2275] restore
2024-04-22 10:26:43,827 INFO    MainThread:8568 [wandb_run.py:_restore():2281] restore done
2024-04-22 10:26:51,710 INFO    MainThread:8568 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-22 10:26:51,710 INFO    MainThread:8568 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-22 10:26:51,712 INFO    MainThread:8568 [wandb_run.py:_footer_sync_info():3879] logging synced files
